doc/detection/Footprint.rst

```diff
@@ -0,0 +1,130 @@
+=========
+Footprint
+=========
+
+The fundamental unit of a detection in the LSST pipeline is an instance of the
+``Footprint`` class. This class contains the x,y location for pixels which are
```

Comment:
I am not sure what you are asking here, yes it is only the intensity of the peask